packages/maskbook/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "outDir": "./dist/",
     "tsBuildInfoFile": "./dist/.tsbuildinfo"
   },
-  "include": ["./src/**/*.tsx", "./src/**/*.ts", "./src/**/*.json"],
+  "include": ["./src/**/*.tsx", "./src/**/*.ts", "./src/**/*.json", "src/plugins/Augur/icons.tss"],
```

Comment:
please revert this change. it seems like an error.

Reply:
Reverted.